examples/platform/silabs/efr32/rs911x/hal/rsi_board_configuration.h

```diff
@@ -35,7 +35,8 @@ typedef struct
 #elif defined(EFR32MG24_BRD4187C) || defined(BRD4187C)
 #include "brd4187c.h"
 #else
-#error "Need SPI Pins"
+#include "sl_custom_board.h"
+#warning "SPIDRV USART peripheral not configured correctly"
```

Comment:
Is this warning message valid? Why do we allow this path? I could not find the content of sl_custom_board.h. How do we know that peripheral isn't correctly configured in in the sl_custom_board

Reply:
Updated the warning message